src/patternfly/components/Pagination/pagination.scss

```diff
@@ -104,13 +107,12 @@ $pf-v6-c-pagination--variable-map: build-spacer-map("none", "sm", "md", "lg", "x
 
     .#{$pagination}__page-menu {
       position: absolute;
-      display: block;
+      display: var(--#{$pagination}--m-bottom__page-menu--Display);
     }
 
     .#{$pagination}__nav {
       display: flex;
       flex-basis: 100%;
-      justify-content: space-between;
```

Comment:
This was causing an issue where visual focus wasn't matching with tab order. The following screenshot makes it seems like pressing Tab would make the "previous" button on the left be focused first, then the menu toggle, but the menu toggle receives focus first then the "previous" button does. I can update this styling further (do we want it to look more like compact pagination where everything is pushed to the right, or do we want all controls centered?), but just wanted to remove this for now to at least fix the issue.

![Bottom Pagination at breakpoint, showing a visual order that doesn't match with the actual tab order](https://github.com/patternfly/patternfly/assets/70952936/8cddd3cc-1f46-411b-bb6f-95af844b624a)

Reply:
Hmm.... I don't see a specific design in figma for bottom pagination on mobile - did you talk with design about this change? If not, IMO this is probably better as a separate issue to address the a11y of the tab order and update the design/layout.

For reference, this is what it looks like now
<img width="528" alt="Screenshot 2024-05-22 at 5 31 22 PM" src="https://github.com/patternfly/patternfly/assets/35148959/08a1ec35-87f8-4288-887e-bb86eb4c520f">

If we go with something like this, it's probably worth looking at the CSS some more to see if we can not use absolute positioning for the menu since the way that's written currently is specific to the old layout.

Reply:
@kaylachumley @lboehling wdyt? My screenshot is the existing design, Michael's screenshot is what it looks like with my updates in this PR. If we'd want something different from what is in this PR I can revert and open a followup to tackle this.

Reply:
I think the first example is a better interaction for mobile use -- separating the front and back arrows more is better for touch interaction/a more typical layout for mobile!

Reply:
@thatblindgeye other than this comment, the PR looks good to merge 👍

Reply:
Opened https://github.com/patternfly/patternfly/issues/6731 as a followup